src/electron/electron/config/logger.config.ts

```diff
@@ -9,5 +9,5 @@ export const LOG_FILE_NAME = {
 // see: https://github.com/megahertz/electron-log#log-levels
 export const LOG_LEVEL = {
   CONSOLE: 'silly' as LevelOption,
-  FILE: 'silly' as LevelOption
+  FILE: 'info' as LevelOption
```

Comment:
can you do the `is.dev` check here so you don't have the `LevelOption`'s  (silly and info) in the same place?

Reply:
done, thanks for the suggestion. please re-review :)